lib/chef/provider/systemd_unit.rb

```diff
@@ -33,7 +33,7 @@ class SystemdUnit < Chef::Provider
       provides :systemd_unit, os: "linux"
 
       def load_current_resource
-        @current_resource = Chef::Resource::SystemdUnit.new(new_resource.name)
```

Comment:
This one should actually be `.name` because that's what it is used for :) (only rarely matters, usually for logging)

Reply:
👍